rest_framework/authtoken/models.py

```diff
@@ -51,4 +51,5 @@ def pk(self):
     class Meta:
         proxy = 'rest_framework.authtoken' in settings.INSTALLED_APPS
         abstract = 'rest_framework.authtoken' not in settings.INSTALLED_APPS
-        verbose_name = "token"
+        verbose_name = _("Token")
```

Comment:
just to make sure, adding this two lines with modified values, is there any possible missing migrations?

Reply:
As far as I understand it, no. Migrations only accept hard coded values, but since this is not one, a migration would not make sense. It must be created when the language is set.

Reply:
the TokenProxy model is inheriting from https://github.com/encode/django-rest-framework/blob/master/rest_framework/authtoken/models.py#L27 but still not showing it properly, right?

Reply:
from the migrations history https://github.com/encode/django-rest-framework/tree/master/rest_framework/authtoken/migrations it would be safer to check locally if new migrations are generated with this two attribute changes in meta. if not it is OK. but if so, we must have to ship it with the package :)

Reply:
thanks for adding the missing auto migration